x-pack/plugin/transform/qa/single-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/transform/integration/TransformTaskFailedStateIT.java

```diff
@@ -84,10 +84,10 @@ public void testForceStopFailedTransform() throws Exception {
         assertThat(ex.getResponse().getStatusLine().getStatusCode(), equalTo(RestStatus.CONFLICT.getStatus()));
         assertThat(
             (String) XContentMapValues.extractValue("error.reason", entityAsMap(ex.getResponse())),
-            matchesRegex(
-                "Unable to stop transform \\[test-force-stop-failed-transform\\] as it is in a failed state with reason \\["
+            startsWith(
```

Comment:
BTW, could it be `equalTo` now?

Reply:
It cannot, there's a much longer error message behind it: `Failed to index documents into destination index due to permanent error: [org.elasticsearch.xpack.transform.transforms.BulkIndexingException: Bulk index experienced [7] failures and at least 1 irrecoverable [org.elasticsearch.xpack.transform.transforms.TransformException: Destination index mappings are incompatible with the transform configuration.; org.elasticsearch.index.mapper.DocumentParsingException: [1:30] failed to parse field [reviewer] of type [long] in document with id 'dRQkp_t5ubOfpvJaH8A0aekAAAAAAAAA'. Preview of field's value: 'user_0'; java.lang.IllegalArgumentException: For input string: "user_0"].; org.elasticsearch.xpack.transform.transforms.TransformException: Destination index mappings are incompatible with the transform configuration.; org.elasticsearch.index.mapper.DocumentParsingException: [1:30] failed to parse field [reviewer] of type [long] in document with id 'dRQkp_t5ubOfpvJaH8A0aekAAAAAAAAA'. Preview of field's value: 'user_0'; java.lang.IllegalArgumentException: For input string: "user_0"]`

It's possible to be `equalTo` this but I can't seem to figure out why we switched from that to `matchRegex`